prefect_gcp/cloud_storage.py

```diff
@@ -7,6 +7,7 @@
 from typing import Any, BinaryIO, Dict, List, Optional, Tuple, Union
 from uuid import uuid4
 
+import pandas as pd
```

Comment:
We probably don't want to add this as a requirement to keep this package lightweight. I recommend doing try/except ImportError.

Reply:
Makes sense! Working on it!

Reply:
No rush, but thanks for being responsive and open to suggestions!

Reply:
Fixed 👍🏻